### DISCUSSION TITLE
is there any way to do this on a windows 11 school laptop

### DISCUSSION THREAD

**Discussion by Tagics** [3kh0/ext-remover#606] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
it doesnt work

**Comment by cooltoo37** [comment, marked as answer]
dayum windows 11 ur school hella rich

**Reply by Aka-but-nice** [reply]
> it doesnt work

What extension?

**Reply by RapidOnGfuel** [reply]
wait version of windows?
education?

**Comment by AnOrangeCone** [comment]
bro at that point make a bootable windows 10/11 drive and install a fresh clean install with no restrictions and if the bios is fucked well flash a new bios

**Reply by AshtonDavies** [reply]
Windows education does have booting restrictions and it is pretty secure, although I have seen a YouTuber hack it before.
